example/typescript/typescript-example.ts

```
@@ -48,80 +48,97 @@ const htmlString = `<!DOCTYPE html>
-    console.log(`${docType} document created: ${fileName}`);
+      }
+      switch (docResult.constructor?.name) {
+        case 'Buffer':
+          docData = docResult as Buffer;
+          break;
+        case 'ArrayBuffer':
+          docData = Buffer.from(docResult as ArrayBuffer);
+          break;
+        case 'Blob':
+          console.log(`Received Blob for ${docType}, converting to ArrayBuffer then Buffer...`);
+          const arrayBuffer = await (docResult as Blob).arrayBuffer();
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)